sklearn/cluster/hierarchical.py

```diff
@@ -814,10 +823,10 @@ def fit(self, X, y=None):
         if self.linkage != 'ward':
             kwargs['linkage'] = self.linkage
             kwargs['affinity'] = self.affinity
-        self.children_, self.n_components_, self.n_leaves_, parents = \
-            memory.cache(tree_builder)(X, connectivity,
-                                       n_clusters=n_clusters,
-                                       **kwargs)
+        self.children_, self.n_connected_components_, self.n_leaves_, \
```

Comment:
I'd be better if you put parentheses around this list of variables, and put the line break in the middle of that list. We try to avoid line continuation with \

Reply:
Thanks for your feedback Joel, this should be solved by commit #5d30766